src/components/container/Container.tsx

```diff
@@ -13,7 +13,7 @@ const StyledContainer = styled.div`
 export class Container extends React.Component {
   render() {
     return (
-      <StyledContainer>
+      <StyledContainer className={"container"}>
```

Comment:
Waarom enkel typografie beschikbaar maken voor elementen die in .container zitten?